backend/graminsta/post/urls.py

```diff
@@ -8,5 +8,5 @@
 
 app_name = 'post'
 urlpatterns = [
-    path('createPost/', PostRecordView.as_view(), name='post'),
+    path('create-post/', PostRecordView.as_view(), name='post'),
```

Comment:
I think in your case, RESTful API `/<prefix>/post` can fit the requirements.

```
GET /<prefix>/post   Retrieve list of Posts
GET /<prefix>/post/5  Retrieve Post whose id is 5
POSt /<prefix>/post    Create a Post
```

Then your url endpoints need to be updated.